experimental/PropertyDDS/packages/property-inspector-table/src/ShareModal.tsx

```diff
@@ -78,15 +78,25 @@ export interface IShareModalState {
 	};
 }
 
+export type SharingPermission = "read" | "write" | "delete";
+
 export interface IShareOptions {
 	/**
 	 * The handler that is invoked if the user decides to share.
 	 */
-	shareHandler: (userIds: string[], groupIds: string[]) => Promise<any>;
+	shareHandler: (
+		userIds: string[],
+		groupIds: string[],
+		permissions: { actions: SharingPermission[] },
+	) => Promise<any>;
 	/**
 	 * The handler that is invoked if the user decides to unshare.
 	 */
-	unshareHandler: (userIds: string[], groupIds: string[]) => Promise<any>;
+	unshareHandler: (
+		userIds: string[],
+		groupIds: string[],
+		permissions: { actions: SharingPermission[] },
+	) => Promise<any>;
```

Comment:
TypeScript 5 disallows passing an extra undeclared 3rd argument to the share/unshare callbacks.

I only found one reference to each callback, which passed '{ actions: ["read", "write", "delete"] }'.  I've added the necessary signature.